rasa/core/test.py

```diff
@@ -914,6 +967,22 @@ async def _collect_story_predictions(
     )
 
 
+def _filter_step_events(step: StoryStep) -> StoryStep:
+    events = []
+    for event in step.events:
+        if (
+            type(event) == WronglyPredictedAction
```

Comment:
Can we use `isinstance(event, WronglyPredictedAction)` to be consistent with the rest of the codebase?